src/components/SitewideHeader/DocsDrawerMenu.js

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react';
+import { Popover, Transition } from '@headlessui/react'
+
+import DrawerMenuHeader from './DrawerMenuHeader';
+import MobileDropdownNavItem from './MobileDropdownNavItem';
+import { DOCS_LAYOUTS } from 'components/doc';
+
+const subItems = DOCS_LAYOUTS.map((props) => ({
+  name: props.tabLabel,
+  to: props.startPath,
+  isActiveMatch: props.isActiveMatch,
+}));
```

Comment:
The drawer menu for the docs now just shows the docs categories so it needs to be different than the drawer menu for the rest of the site.